uiowa/uiowa01
# Feature: Breakpoints and Template fixes

Addresses issues #268, #280, and #276.

# How to test

1. Run `drush @theming.local si collegiate`
2. Create a basic page of content.
3. For #268, Verify that header, footer, and content area container margins look the same as the browser collapses to a smaller size.
4. For #280, Adjust the theme menu settings so that the main menu appears below the site name and verify that there is no extra padding being added when the browser collapses.
5. For #276, Add an empty menu with the same theme setting as above.  Verify that if the menu block is removed from the primary_menu region on the blocks page that no underline still appears below the site title.   **Note: if an empty menu is still placed within the primary_menu block section, the underline will still appear.**
